tsconfig.samVisualize.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "outDir": "./samVisualizeRenderJS",
+        "removeComments": true
+    },
+    "include": ["./src/samVisualize/rendering/*.ts"]
```

Comment:
samVisualize is part of the toolkit source, so it doesn't need a separate tsconfig